### DISCUSSION TITLE
使用paddle-ocr和PyMuPDF识别PDF文件的问题

### DISCUSSION THREAD

**Discussion by AHPUymhd** [PaddlePaddle/PaddleOCR#12729] [Q&A]
#### 运行环境 / Runtime Environment
- OS:
- Paddle:
- PaddleOCR:

#### 复现代码 / Reproduction Code
在我的代码中可以将一个PDF转成图片然后进行识别文本，但是我将他用pyinstaller打包成exe的时候，这个exe运行 时会报错，说找不到fitz这个模块，但是打包之前的项目是可以运行的，也不会出现这个问题
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/72792297/c613fd34-c944-463a-8b1a-6e5bb2c7f46a)
python：3.11
paddle-ocr:2.7.0.3
paddle-gpu:2.6.0

#### 完整报错 / Complete Error Message

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/72792297/cd420cc0-ea9d-4fc6-bce9-379086e37f3b)

**Comment by AHPUymhd** [comment]
@GreatV 可以请您解答一下吗，谢谢了

**Comment by GreatV** [comment]
试试 “打开生成的 .spec 文件，添加 fitz 模块的依赖项。在 hiddenimports 列表中添加 fitz 模块。然后重新打包。”

**Comment by AHPUymhd** [comment]
但是打包完了还是一样的问题怎么办




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PaddlePaddle/PaddleOCR"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月3日(星期一) 下午4:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [PaddlePaddle/PaddleOCR] 使用paddle-ocr和PyMuPDF识别PDF文件的问题 (Issue #12607)





 
试试 “打开生成的 .spec 文件，添加 fitz 模块的依赖项。在 hiddenimports 列表中添加 fitz 模块。然后重新打包。”

**Comment by GreatV** [comment]
参考一下这里： https://stackoverflow.com/questions/25733467/no-module-named-when-using-pyinstaller
